lib/matplotlib/axes/_axes.py

```diff
@@ -3828,48 +3829,48 @@ def scatter(self, x, y, s=None, c=None, marker=None, cmap=None, norm=None,
 
         marker : `~matplotlib.markers.MarkerStyle`, optional, default: 'o'
             See `~matplotlib.markers` for more information on the different
-            styles of markers scatter supports. `marker` can be either
+            styles of markers scatter supports. *marker* can be either
             an instance of the class or the text shorthand for a particular
             marker.
 
         cmap : `~matplotlib.colors.Colormap`, optional, default: None
             A `~matplotlib.colors.Colormap` instance or registered name.
-            `cmap` is only used if `c` is an array of floats. If None,
-            defaults to rc `image.cmap`.
+            *cmap* is only used if *c* is an array of floats. If ``None``,
+            defaults to rc ``image.cmap``.
 
         norm : `~matplotlib.colors.Normalize`, optional, default: None
             A `~matplotlib.colors.Normalize` instance is used to scale
-            luminance data to 0, 1. `norm` is only used if `c` is an array of
-            floats. If `None`, use the default :func:`normalize`.
+            luminance data to 0, 1. *norm* is only used if *c* is an array of
+            floats. If ``None``, use the default :func:`normalize`.
```

Comment:
I think we should fix this.  What is `normalize`?  I think they mean it goes to `colors.Normalize`